CMakeLists.txt

```diff
@@ -3,19 +3,25 @@ project(lvgl)
 set(CMAKE_C_STANDARD 11)#C11
 set(CMAKE_CXX_STANDARD 17)#C17
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_C_FLAGS "-Wno-error=int-conversion")
 
 INCLUDE_DIRECTORIES(${PROJECT_SOURCE_DIR})
 
 file(GLOB_RECURSE INCLUDES "./*.h" )
 
+# Here is only an example to use static lib. If you use shared lib,
+# simply change SDL2-static to SDL2. Set SDL2_DIR to where SDL2Config.cmake
+# exists in the SDL2 package.
+set(SDL2_DIR "C:/Users/Administrator/Desktop/SDL/lib/lib/cmake/SDL2")
```

Comment:
I think we shouldn't hard code such a thing. Instead we can add to the README how to set `SDL2_DIR` when calling cmake.

Reply:
If you use an installer, SDL2 normally will be installed in some system folder. In this case SDL2 can be easily found by find_package() in module mode without setting SDL2_DIR.
However I prefer to compile the source code and install the SDL2 lib in some managed directory (It's a bad example here as I install the lib at the Desktop). In this case I have to specify SDL2_DIR explicitly before calling find_package(... CONFIG). If later I don't want to use SDL2, I delete the folder leaving my system clean.
It depends on your preference.

Reply:
If you use apt to install SDL2 for example, both shared and static lib will be installed. You can instruct CMake to use the specified lib by you. SDL2 means shared lib while SDL2-static means static lib.

Reply:
I think we should have a config which works out of the box for the most common scenarios, but make it open for customization. 


So although I'm not an experienced CMake user (so correct me if it doesn't work for any reason), I think `find_package()` would be better by default, but we should allow passing the path as a CMake parameter too.